doc/releases/changelog-dev.md

```diff
@@ -93,6 +93,10 @@
   a sparse matrix.
   [(#6173)](https://github.com/PennyLaneAI/pennylane/pull/6173)
 
+* The `make_plxpr` function is added, to take a `QNode` and create a `Callable` that,
+  when called, will return a PLxPR representation of the `QNode`.
```

Comment:
This sounds like `make_plxpr` can only be used on `QNode`s, is that the plan?
This would be a major difference to `jax.make_jaxpr`, which is also used extensively on quantum functions that are not QNodes, on hybrid functions that contain but aren't QNodes, and even on purely classical ones.

Unrelatedly, maybe?
```suggestion
* The `capture.make_plxpr` function is added, to take a `QNode` and create a `Callable` that,
  when called, will return a PLxPR representation of the `QNode`.
```

Reply:
Yep, `make_plxpr` should work just like `make_jaxpr` --- on arbitrary functions that may/may not include QNodes

Reply:
Then I suggest to update the signature and the docs :)

Reply:
updated :)